perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ConsentsManagerBlImpl.java

```diff
@@ -91,4 +94,17 @@ public ConsentHub updateConsentHub(PerunSession perunSession, ConsentHub consent
 			throw new ConsistencyErrorException("Updated consentHub " + consentHub + " was not found!");
 		}
 	}
+
+	@Override
+	public void addFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws FacilityAlreadyAssigned {
+		getConsentsManagerImpl().addFacility(sess, consentHub, facility);
+	}
+
+	@Override
+	public void removeFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws RelationNotExistsException, ConsentHubAlreadyRemovedException {
```

Comment:
Now we can use this method in `FacilitiesManagerBlImpl.deleteFacility` - there is _TODO_ from #3524.